src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -254,12 +236,35 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
         return [DATA_IMPORT_PAYMENT_AUTHORIZATION_TOKEN_FIELD.fieldApiName];
     }
 
+    /**
+     * @description Sets name on the card based on the donor selection
+     */
     @api
     setNameOnCard(cardHolderName) {
         this.cardHolderName = cardHolderName;
     }
 
-    dispatchApplicationEvent (eventName, payload) {
+    /**
+     * @description Returns the cardholder name.
+     * If the Donor Type = Account, the first and last name are empty.
+     * If the Donor Type = Contact, the Contact first name (if specified) 
+     * and the last name are provided to the Payments Services card tokenization iframe.
+     */
+    getCardholderName() {
```

Comment:
The cardholder name is passed to the card tokenization process. It is constructed with the contact's first name (if the contact has it) and the last name if the Donor Type = Contact. If the Donor Type = Account, an empty cardholder name is passed to the tokenization process with used "Not provided" as default for the card tokenization.